chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -415,10 +430,16 @@ bool BraveRenderViewContextMenu::IsCommandIdEnabled(int id) const {
     case IDC_AI_CHAT_CONTEXT_REWRITE:
       return ai_rewriter::features::IsAIRewriterEnabled();
 #endif
+
+    case IDC_CONTENT_CONTEXT_OPENLINK_SPLIT_VIEW:
+      return !IsInProgressiveWebApp() &&
```

Comment:
I think we should check if browser->is_type_normal(), so that this won't be populated in popup window.

Reply:
It looks like `CanSplitViewEnabled()` can check if the browser is type_normal or not. Then we can simply check

```suggestion
      return CanSplitViewEnabled(...)
```